src/view3d.cpp

```diff
@@ -311,37 +461,42 @@ void View3D::paintGL()
 //        s[barSize] = '|';
 //    tfm::printfln("%12f %4d %s", quality, frameTime, s);
 
-    m_incrementalFramebuffer->release();
-    QGLFramebufferObject::blitFramebuffer(0, QRect(0,0,width(),height()),
-                                          m_incrementalFramebuffer.get(),
-                                          QRect(0,0,width(),height()),
-                                          GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    // TODO: this should really render a texture onto a quad and not use glBlitFramebuffer
```

Comment:
Why?  In principle `glBlitFramebuffer` should be strictly faster, the intention is to trigger a simple `memcpy()` of device memory.  Is it slower in practice because the driver writers don't optimize this path?